ui/shared/components/panel/variants/VariantClassify.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button, Label } from 'semantic-ui-react'
+import { CopyToClipboard } from 'react-copy-to-clipboard'
```

Comment:
I think we want to use `React.lazy` for this library, as this is the only place it is used and therefore we can avoid adding it to our main JS bundle. Alternatively, we could move the`ClinGenAlleleId` component to its own file and lazily load it here